bims/api_views/location_site.py

```diff
@@ -289,6 +294,138 @@ def get(self, request):
             return Response(response_data)
 
 
+    def get_biodiversity_data(self, collections):
```

Comment:
Usually we should avoid 'get_' prefixes for methods - it is verbose and does not add meaning...

Reply:
Also we should have docstrings explaining what parameters are and what their datatypes are.

Reply:
I think it’s partly our (old developer) fault, we use get_ prefix for some methods. So I think Christiaan did this for the consistency.

Reply:
ok no problem